prow/config/presubmits.yaml

```diff
@@ -1245,6 +1245,55 @@ presubmits:
       - emptyDir: {}
         name: varlibdocker
 
+  - name: istio-push-images-2.3
+    trigger: (?m)^/test( | .* )push\-images,?($|\s.*)
+    rerun_command: /test push-images
+    skip_report: false
+    max_concurrency: 2
+    always_run: true
```

Comment:
This can be set to false so it does not run automatically on every pr. One should comment /test <name> to run it

Reply:
No, the whole point is to push the images for every PR, so that when someone wants to review it, they can immediately use the images. If they have to comment and then wait for the images to be built, they might as well build and push them locally.

Reply:
+1 on the trivial images, but this then opens up the following possibilities:
- PR author forgets to build the image -> no benefit for reviewer
- PR author updates PR, but forgets to build the images -> reviewer (and author) use an out-of-date image, wasting time before realizing what's wrong

But I'll set this to `false` for now, while we test this. Later, we should probably add a label(?) for trivial PRs that causes this job to be skipped.